nemo_text_processing/inverse_text_normalization/zh/taggers/ordinal.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pynini
+from nemo_text_processing.inverse_text_normalization.zh.graph_utils import GraphFst
+from pynini.lib import pynutil
+
+
+# A third way, seems to work fine but might have potential issues?
```

Comment:
what's potential issue?

Reply:
I should've have deleted that one. The comment was for another graph I built way back when I was working on the Ordinal. Nothing is broken. Thanks for pointing out.